Assets/MRTK/Core/Inspectors/Profiles/MixedRealityHandTrackingProfileInspector.cs

```diff
@@ -13,7 +13,8 @@ public class MixedRealityHandTrackingProfileInspector : BaseMixedRealityToolkitC
         private SerializedProperty jointPrefab;
         private SerializedProperty palmPrefab;
         private SerializedProperty fingertipPrefab;
-        private SerializedProperty handMeshPrefab;
```

Comment:
If this is no longer displayed in the inspector, should we remove it from the profile class? Do we see any possible auto-upgrade path?

Reply:
Left it in the base class for auto compat and deprecated the public property. We don't need an entire prefab to power system-provided hand mesh visuals.